examples/notebooks/working_with_registry_results.ipynb

```diff
@@ -1,221 +1,223 @@
+   "metadata": {},
+   "source": [
+    "If you are looking for a particular data collection or catalog, as we\n",
+    "did above when we looked for the NVSS archive, often simply reviewing\n",
+    "the titles is sufficient.  Other times, particularly when you are not\n",
+    "sure what you are looking for, it helps to look deeper.  \n",
+    "\n",
+    "The resource description, available via the \n",
+    "`~pyvo.registry.regtap.ResourceRecord.res_description` property,\n",
+    "tends to be the most revealing.  It contains a paragraph (or two)\n",
+    "summarizing the catalog or data collection.  It will often describe\n",
+    "the scientific intent behind the collection.  \n",
+    "\n",
+    "The `~pyvo.registry.regtap.RegistryResource.short_name` can also be\n",
+    "helpful, as well.  This name is meant to be short--16 characters or\n",
+    "fewer; consequently, the value is often includes the abbreviation for the\n",
+    "project or observatory that produced the collection or catalog.  \n",
+    "\n",
+    "A selection of the resource metadata, including the title, shortname and\n",
+    "desription, can be printed out in a summary form with\n",
+    "the `~pyvo.registry.regtap.RegistryResource.describe` function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nvss.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Registry Records can be converted to a service object"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nvss = colls[0].service  # converts record to service object\n",
+    "print(nvss.baseurl)\n",
+    "query = nvss.create_query(size=0.25, format=\"image/fits\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Thus, not only does this service instance contain the base URL but it\n",
+    "also includes all of the metadata from the registry that desribes the\n",
+    "service.  With this service object, we can either call its \n",
+    "`~pyvo.dal.sia.SIAService.search` function directly or \n",
+    "create query objects to get cutouts for a whole list of sources."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Our discussion of service metadata offers an opportunity to highlight\n",
+    "another important property, the service's *IVOA Identifier* (sometimes\n",
+    "referred to as its *ivoid*).  This is a globally-unique identifier\n",
+    "that takes the form of a \n",
+    "`URI <http://en.wikipedia.org/wiki/Uniform_resource_identifier>`_:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "colls = vo.regsearch(keywords=[\"NVSS\"], servicetype='sia')\n",
+    "for coll in colls:\n",
+    "    print(coll.ivoid)"
```

Comment:
Let me know if I missed any other notebook changes that I missed.

Reply:
Yeah, I was wondering if there was a magic way to get it to diff better.  The second change was in the last cell that originally had ([see it on nbviewer](https://nbviewer.jupyter.org/github/astropy/pyvo/blob/master/examples/notebooks/working_with_registry_results.ipynb)):

```
nvss = services[0]                             # RegistryResource
nvsss = nvss.service                           # SIAService
nq = nvss.create_query(pos=(350.85, 58.815),
                      size=0.25, 
                      format="image/fits")     # SIAQuery
```
where either the second line with "nvsss" has too many 's's or the third line should have used the "nvsss".  I'm not sure which was intended.  That's the only place the SIAService object is used, so it could be fixed either way.  I originally assumed that the 'sss' was a typo, but maybe it's for "service" and I should instead add the extra 's' to the next line.

Reply:
On reflection, I changed my mind and kept the "nvsss" object, correcting the last line above, and adding a print statement to show the type of that object, which I think must have been the original intention.

Reply:
Okay and just to make completely sure, this didn't work before because it was calling on the wrong variable?

My suggestion would be for this one to just change the names to make it clearer what is going on, and get rid of the comment parts

Reply:
Yes, it did not work the way it was yesterday:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-6508e30a21a4> in <module>
      3 nvss = services[0]                             # RegistryResource
      4 nvsss = nvss.service                           # SIAService
----> 5 nq = nvss.create_query(pos=(350.85, 58.815),
      6                       size=0.25,
      7                       format="image/fits")     # SIAQuery

AttributeError: 'RegistryResource' object has no attribute 'create_query'
```
Yes, I can name them more clearly.

Reply:
Actually, now that I start to look at the notebook contents with an eye toward improving it rather than just getting it running, there are a couple more problems.  Stay tuned.

Reply:
Does anybody know why the notebook uses the two step process of creating a SIAQuery object and then executing it?  I'd say we simple show people to call the SIAService's search() function.

Reply:
The bane of having two ways, there's always that other way :) I'm fine with either!  Thanks for taking the time to improve the notebook, improving examples is always time well spent!

Reply:
I just clarified the two ways in the notebook.